docker/test_dockerimage.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+which asm2wasm
+which llvm-ar
+which emsdk
+node --version
+npm --version
+python --version
+pip --version
+python3 --version
+pip3 --version
```

Comment:
Do we need python3, pip3, or pip?  There is no point if having 3 different version of python installed is there?

Reply:
You mean two versions I believe, as there is no Python installed with Emscripten SDK. 
Motivation to have python3 comes from https://github.com/WebAssembly/binaryen/pull/2281 
About python2 - I don't have strong feelings. Maybe just that it takes a time for projects to migrate, and my intention was to provide a complete container that can be used in a general case. 

I can remove python2 in exchange of documentation how to extend this image.

Reply:
Had to revert that. 
`em++` Still has a header `#!/usr/bin/env python` and I think making `python` pointing to `python3` is a bad way to mitigate this issue.

Reply:
OK.